.github/workflows/coverity.yml

```diff
@@ -28,33 +21,29 @@ jobs:
           fetch-depth: 0
           submodules: recursive
       - uses: ./.github/actions/environment
-      - name: test
-        run: make -f .test.mk test-coverity
-        env:
-          COVERITY_TOKEN: ${{ secrets.COVERITY_TOKEN }}
+
+      - name: Determine tarantool version
+        run: printf 'version=%s\n' $(git describe --long --always HEAD) >> "${GITHUB_OUTPUT}"
+        id: version
+
+      # Not a full list of dependencies. Just ones that are
+      # required to successful configuration stage (cmake) and
+      # missed in the runner's environment.
+      - name: Setup tarantool dependencies
+        run: sudo apt install -y libreadline-dev
+
+      - name: Configure
+        run: cmake -S . -B . -DCMAKE_BUILD_TYPE=RelWithDebInfo -DENABLE_WERROR=ON -DTEST_BUILD=ON
+
+      - uses: vapier/coverity-scan-action@v1
```

Comment:
Thanks for noticing! I'm aware of these recipes, but I found it nice that there is a place, where it is supported by the community. If we have problems with it, I would prefer to send a patch to the action -- this way it may be useful for others too.